lib/types.ts

```diff
@@ -18,6 +18,7 @@ export type ApiPost = {
     spanish_mp3_url: string;
     spanish_audio_size: number;
     spanish_audio_duration: number;
+    category: undefined | { key: 'teaching'; value: 'Teaching' };
```

Comment:
i think this should actually be:

```suggestion
    category: undefined | { key: 'teaching'; value: 'Teaching' } | { key: 'post'; value: 'Post' };
```